src/components/AddBookForm/form.css

```diff
@@ -0,0 +1,60 @@
+.form-heading {
+  margin-left: 40px;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+#title {
+  width: 33%;
+  height: 35px;
+  padding: 5px;
+  border-radius: 4px;
+  border: solid 1px #ffff;
+  background-color: #ffff;
+}
+
+#author {
+  width: 33%;
+  padding: 5px;
+  height: 35px;
+  border-radius: 4px;
+  border: solid 1px #ffff;
+  background-color: #ffff;
+}
+
+#select {
+  height: 35px;
+  border-radius: 4px;
+  border: solid 1px #ffff;
+  background-color: #ffff;
+}
+
+form {
+  width: 95%;
+  margin: 5px 40px;
+  display: flex;
+  gap: 3%;
+}
+
+.add-btn {
+  height: 35px;
+  padding: 4px;
+  border: none;
+  cursor: pointer;
+  border-radius: 3px;
+  width: max-content;
+  background-color: #0290ff;
+  color: #ffff;
+}
```

Comment:
- [ ] Kindly make sure to follow the button width, spacing on the right and left side of the page as shown in the video.

https://user-images.githubusercontent.com/70269507/173210649-2955c00f-5c3d-4e8b-8195-95d952b4f96f.mov

- [ ] Kindly make sure to display the progress bar in a straight line as per the design. It should be aligned properly.

https://user-images.githubusercontent.com/70269507/173210647-13cc633c-3a76-45dd-adbe-c0a7d2bc193b.mov